<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dice</class>
 <widget class="QDialog" name="dice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>851</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="enemyDicesLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>540</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>770</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyj≈õcie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>660</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>840</y>
     <width>851</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="playerDicesLayout"/>
  </widget>
  <widget class="QLabel" name="lblResult">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>650</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rezultat</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDice_1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>790</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDice_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>790</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDice_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>790</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDice_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>790</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDice_5">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>790</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReroll">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>790</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Przelosuj wybrane</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
